core/internal/evaluator/caching.go

```diff
@@ -324,12 +324,24 @@ func calculatePartitionStatus(offsets []*protocol.ConsumerOffset, brokerOffsets
 			return protocol.StatusStop
 		}
 
+		// Its possible the consumer had a rewind in the interval, though it has some lag currently.
+		// We only count a rewind state against the consumer for as long as it it rewinds (and recovery) occurs. For
```

Comment:
the statement doesn't read well. Did you mean
"We dont count a rewind state against the consumer if it rewinds and recovers immediately."